data/mods/Aftershock/recipes/appliance_recipes.json

```diff
@@ -0,0 +1,93 @@
+[
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "afs_battery_charger",
+    "category": "CC_APPLIANCE",
+    "subcategory": "CSC_APPLIANCE_UTILITY",
+    "skill_used": "electronics",
+    "difficulty": 2,
+    "skills_required": [ [ "fabrication", 2 ] ],
+    "time": "45 m",
+    "reversible": true,
+    "autolearn": [ [ "electronics", 2 ] ],
+    "using": [ [ "soldering_standard", 5 ] ],
+    "qualities": [ { "id": "SCREW", "level": 1 } ],
+    "components": [
+      [ [ "afs_energy_storage_2", 1 ] ],
+      [ [ "afs_energy_storage_1", 2 ] ],
+      [ [ "afs_circuitry_2", 1 ] ],
+      [ [ "afs_material_1", 5 ] ],
+      [ [ "cable", 10 ] ]
+    ]
+  },
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "afs_survival_heater",
+    "category": "CC_APPLIANCE",
+    "subcategory": "CSC_APPLIANCE_UTILITY",
+    "skill_used": "electronics",
+    "difficulty": 3,
+    "skills_required": [ [ "fabrication", 4 ] ],
+    "time": "2 h 30 m",
+    "reversible": true,
+    "autolearn": [ [ "electronics", 3 ] ],
+    "using": [ [ "soldering_standard", 10 ] ],
+    "qualities": [
+      { "id": "HAMMER", "level": 2 },
+      { "id": "SAW_M", "level": 1 },
+      { "id": "SCREW", "level": 1 },
+      { "id": "WRENCH", "level": 1 }
+    ],
+    "components": [
+      [ [ "afs_heat_2", 2 ] ],
+      [ [ "afs_heat_1", 3 ] ],
+      [ [ "afs_circuitry_2", 1 ] ],
+      [ [ "afs_material_2", 5 ] ],
+      [ [ "cable", 15 ] ]
```

Comment:
Could probably do with a large amount of heating elements instead of heat_2, also sheet metal for the frame too.